src/FhirClientProvider.tsx

```diff
@@ -40,64 +41,90 @@ export default function FhirClientProvider(props: Props): JSX.Element {
         });
     }
 
-    async function getCarePlan(client: Client, patientId: string): Promise<CarePlan> {
-        if (!client) return;
-        let params = new URLSearchParams({
-            "subject": `Patient/${patientId}`,
-            "category": IsaccCarePlanCategory.isaccMessagePlan.code,
-            "status": "active",
-            "_sort": "-_lastUpdated"
-        }).toString();
-        return await client.request(`/CarePlan?${params}`).then((bundle: Bundle) => {
-            if (bundle.type === "searchset") {
-                if (!bundle.entry) {
-                    return null;
-                }
-                if (bundle.total > 1) {
-                    console.log("Multiple ISACC CarePlans found. Using the most recently updated.", bundle);
-                }
-                let firstResult = bundle.entry[0].resource;
-                if (firstResult.resourceType === "CarePlan") {
-                    return CarePlan.from(bundle.entry[0].resource as ICarePlan);
-                } else {
-                    setError("Unexpected resource type returned");
-                    return null;
-                }
-            } else {
-                setError("Unexpected bundle type returned");
-                return null;
+    async function getCarePlans(
+      client: Client,
+      patientId: string,
+      params: object = null
```

Comment:
what is this for? doesn't look like the new parameter is ever used?

Reply:
@hannahburkhardt thank you for catching it, I will remove.  I think it is remnant during my process of refactoring the code.